Develop

smithjustinm opened the pull request:
Added two python scripts for cryptocompare API

smithjustinm committed:
Update .gitignore

smithjustinm committed:
Added python scripts for API calls

scientistkev commented on coin_info.py:
```diff
@@ -0,0 +1,42 @@
+# write to CSV
+with open("coin_info.csv", mode="w") as test_file:
+    test_file_writer = csv.writer(
+        test_file, delimiter=",", quotechar='"', quoting=csv.QUOTE_NONNUMERIC
```
To write a header row, insert the argument: `csv.writer(header=entry)`

scientistkev commented on crypto_data_extraction.py:
```diff
@@ -0,0 +1,212 @@
+        test_file_writer.writerow(entry)
+print("Done getting crypto names and symbols. See coin_info.csv for result")
+
+# Populate BTC prices in USD
```
Let's think about a way to refactor this that will make this more approachable. Maybe writing a recursive function or something...

scientistkev commented on crypto_data_extraction.py:
```diff
@@ -0,0 +1,212 @@
+# write required fields into csv
+with open("btc_prices.csv", mode="a") as test_file:
+    test_file_writer = csv.writer(
+        test_file, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL
```
Don't forget `csv.writer(header=entry)`

scientistkev reviewed (changes requested):
See comments.

smithjustinm committed:
Refactored code with functions for API calls

Added get_coin_data.py with three functions for day, hour, and minute API calls

smithjustinm committed:
Merge pull request #2 from smithjustinm/main

Refactored code with functions for API calls
